File: src/BackEnd/OrderFlow.Business/Models/Item.cs

using OrderFlow.Business.Enums;
using System;

namespace OrderFlow.Business.Models
{
    public class Item : Entity
    {
        public int ProductId { get; set; }
        public Product Product { get; set; }
        public int TableId { get; set; }
        public Table Table { get; set; }
        public int Count { get; set; }
        public decimal Discount { get; set; }
        public bool Delivered { get; set; }

    }


}

File: src/FrontEnd/order-flow/Models/Item.ts
import { CategoryColor, CategoryIcons } from "../constants/Enums";
import { Category } from "./Category";
import { Product } from "./Product";
import { Table } from "./Table";

export class Item {
  id: number = 0;
  productId: number = 0;
  tableId: number = 0;
  product: Product | undefined = undefined;
  table: Table | undefined = undefined;
  Count: number = 0;
  discount: number = 0;
  deliverd: boolean = false;  
}
